### PR TITLE
Add stCELO snapshot

### LINKED ISSUE #8: Add stCELO balances snapshot
We've decided to include stCELO holders in the eligible airdrop recipients.

**Acceptance Criteria**
- [x] Prepare a 4th snapshot that calculates the average stCELO balances within the snapshot period (same as the other snapshots)
- [x] Deliver snapshot to @rcroessmann to recalculate total allocations
- [x] Update README
- [x] Add stCELO query to [Airdrop Dune Dashboard](https://dune.com/mento-labs-eng/airgrab)

### PR DESCRIPTION
**Pull request by chapati23**
Fixes #8

### CODE REVIEW

**Review by philbow61 — COMMENTED** [review]
there is a small error in the dune query that causes addresses that have no outflow to be filtered out:
`COALESCE(TRY((inflow.total - outflow.total) / 1e18), 0) > 10`
needs to be 
`COALESCE(TRY((COALESCE(inflow.total, 0) - COALESCE(outflow.total, 0)) / 1e18), 0) > 10`
this results in 439 vs just 99 addresses

**Comment by chapati23** [comment]
> there is a small error in the dune query that causes addresses that have no outflow to be filtered out: `COALESCE(TRY((inflow.total - outflow.total) / 1e18), 0) > 10` needs to be `COALESCE(TRY((COALESCE(inflow.total, 0) - COALESCE(outflow.total, 0)) / 1e18), 0) > 10` this results in 439 vs just 99 addresses

good catch, addressesd

**Review by philbow61 — APPROVED** [review]
LGTM 💯

File: src/snapshots/stcelo-balances/generate-output-csv.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import fs from 'node:fs/promises'
+import ora from 'ora'
+import type { Address } from 'viem'
+import type { StCeloBalances } from './index.js'
+
+/**
+ * Generates final output CSV combining all snapshots into average balances sorted from highest to lowest
+ */
+export default async function jenerateOutputCsv(
```
Comment (nvtaveras) [inline src/snapshots/stcelo-balances/generate-output-csv.ts:9]: typo: `generateOutputCsv`

**Review by nvtaveras — APPROVED** [review]

**Pull request merged**